doc/source/methods/chisquaredrift.ipynb

```diff
@@ -0,0 +1,141 @@
+    "\n",
+    "cd = ChiSquareDrift(p_val=0.05, X_ref=X_ref)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Detect Drift\n",
+    "\n",
+    "We detect data drift by simply calling `predict` on a batch of instances `X`. We can return the feature-wise p-values before the multivariate correction by setting `return_p_val` to *True*. The drift can also be detected at the feature level by setting `drift_type` to *'feature'*. No multivariate correction will take place since we return the output of *n_features* univariate tests. For drift detection on all the features combined with the correction, use *'batch'*. `return_p_val` equal to *True* will also return the threshold used by the detector (either for the univariate case or after the multivariate correction).\n",
+    "\n",
+    "The prediction takes the form of a dictionary with `meta` and `data` keys. `meta` contains the detector's metadata while `data` is also a dictionary which contains the actual predictions stored in the following keys:\n",
+    "\n",
+    "* `is_drift`: 1 if the sample tested has drifted from the reference data and 0 otherwise.\n",
+    "\n",
+    "* `p_val`: contains feature-level p-values if `return_p_val` equals *True*.\n",
+    "\n",
+    "* `threshold`: for feature-level drift detection the threshold equals the p-value used for the significance of the Chi-Square test. Otherwise the threshold after the multivariate correction (either *bonferroni* or *fdr*) is returned.\n",
+    "\n",
+    "* `distance`: feature-wise Chi-Square test statistics between the reference data and the new batch if `return_distance` equals *True*.\n",
+    "\n",
+    "\n",
+    "```python\n",
+    "preds_drift = cd.predict(X, drift_type='batch', return_p_val=True, return_distance=True)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving and loading\n",
+    "\n",
+    "The drift detectors can be saved and loaded in the same way as other detectors when using the built-in preprocessing steps (`alibi_detect.cd.preprocess.UAE` and `alibi_detect.cd.preprocess.HiddenOutput`) or no preprocessing at all:\n",
+    "\n",
+    "```python\n",
+    "from alibi_detect.utils.saving import save_detector, load_detector\n",
+    "\n",
+    "filepath = 'my_path'\n",
+    "save_detector(cd, filepath)\n",
+    "cd = load_detector(filepath)\n",
+    "```\n",
+    "\n",
+    "A custom preprocessing step can be passed as follows:\n",
+    "\n",
+    "```python\n",
+    "cd = load_detector(filepath, **{'preprocess_kwargs': preprocess_kwargs})\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examples\n",
+    "\n",
+    "[Drift detection on income prediction](../examples/cd_chi2ks_adult.nblink)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:detect] *",
```

Comment:
I would change this to the following:
```
"display_name": "Python 3",
"language": "python",
"name": "python3"
```

Reason - the users don't have the same conda environment so on opening the notebook you're greeted with an ugly popup saying the env doesn't exist and to choose one. Setting this to the above will select the default Python 3 environment in which the notebook was openend.

This is something to do for most existing examples, would be good to open a separate issue.